### PR TITLE
UploadImages

### PR DESCRIPTION
**Pull request by katch00**
the upload works, but it redirects to the backend link for the upload currently.

### CODE REVIEW

File: backend/src/main/java/com/google/capstone/Information.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  private final long id;
+  private final String description;
+  private final long timestamp;
+
+  public Information(long id, String description, long timestamp) {
+    this.id = id;
+    this.description = description;
+    this.timestamp = timestamp;
+  }
+}
```
Comment (isabelsang) [inline backend/src/main/java/com/google/capstone/Information.java:18]: nit: new line

File: backend/src/main/java/com/google/capstone/UploadImg.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+   */
+  @Override
+  protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+      BlobstoreService blobstore = BlobstoreServiceFactory.getBlobstoreService();
+
+      Map<String, List<FileInfo>> uploads = blobstore.getFileInfos(request);
+      List<FileInfo> fileInfos = uploads.get("files");
+  }
+
+}
```
Comment (isabelsang) [inline backend/src/main/java/com/google/capstone/UploadImg.java:63]: nit: new line

File: backend/src/main/java/com/google/capstone/UploadInformation.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
+    String value = request.getParameter(name);
+    if (value == null) {
+      return defaultValue;
+    }
+    return value;
+  }
+
+}
```
Comment (isabelsang) [inline backend/src/main/java/com/google/capstone/UploadInformation.java:92]: same here

File: backend/src/main/java/com/google/capstone/UploadInformation.java [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    mssgEntity.setProperty("description", description);
+    mssgEntity.setProperty("timestamp", timestamp);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(mssgEntity);
+
+    response.sendRedirect("/user-uploads-gallery");
+  }
+
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
```
Comment (BearBurg) [inline backend/src/main/java/com/google/capstone/UploadInformation.java:84]: This method can be static.

**Pull request merged**